build/setup.d/21-install-td-agent-packages.sh

```diff
@@ -0,0 +1,14 @@
+# Ensure that /etc/init.d/td-agent is removed
```

Comment:
Please add at least a shebang. Ideally the script should also fail if something inside fails (even though taupage-init currently does nothing) so we can improve it later.

Reply:
Will do.

Reply:
The script still doesn't fail.